site/_posts/2017-09-25-last-week-of-summer.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Last Week in ManageIQ: Last week of summer ... "
+author: lgalis
+date: 2017-09-26
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+
+This last week of summer found the team back to work on our next release, following an engaging and productive all-hands engineering forum in New Jersey.  
+
+Sprint 69 ended last week, check out the sprint review [slides](https://www.slideshare.net/ManageIQ/sprint-69) and [recording](https://www.youtube.com/watch?v=mQxj_2fNEaI&feature=youtu.be) for all the work completed during this sprint.
+
+
+## Featured
+
+Some statistics for last week:
+  
+31 PRs merged to the [manageiq][manageiq PRs merged] repository,
+
+27 PRs merged to the [manageiq-ui-classic][manageiq-ui-classic PRs merged] repository,
+          
+25 PRs merged to the [manageiq-ui-service][manageiq-ui-service PRs merged] repository,
+
+plus more across the providers' repositories.
+
+
+## Improved
+
+The Greenkeeper bot continues to create PRs to maintain up to date dependencies for the Service UI - see this week's list [here](https://github.com/ManageIQ/manageiq-ui-service/pulls?utf8=✓&q=is%3Apr+is%3Amerged+label%3Agreenkeeper+base%3Amaster+merged%3A"2017-09-18+..+2017-09-24"+sort%3Acreated-desc).
+
+[PR 979](https://github.com/ManageIQ/manageiq-ui-service/pull/979) refactors the CollectionsApi for extensibility (infinite options) and conciseness.<br>
+
+The dialog generation for orchestration template was modified in [PR 15982](https://github.com/ManageIQ/manageiq/pull/15982) to rely on the templates to provide the deployment options.<br> 
+
+[PR 16011](https://github.com/ManageIQ/manageiq/pull/16011) adds 'description' to the list of automate value fields, allowing it to be updated by an automate method.<br> 
+
+## Fixed
+
+To allow the Azure SSA job to succeed, a number of timeouts and workers' memory were increased in [PR 16016](https://github.com/ManageIQ/manageiq/pull/16016).
+
+After [PR 115995](https://github.com/ManageIQ/manageiq/pull/15995), the request to destroy linked events is queued to prevent the UI hanging when deleting an instance of MiqServer.
```

Comment:
minor thing: PR 15995, not 115995 (extra digit)

Reply:
@cybette - fixed